plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -353,14 +353,16 @@ void GSSettingsDlg::UpdateRenderers()
 
 	vector<GSSetting> renderers;
 
-	unsigned renderer_setting = theApp.GetConfig("Renderer", 0);
-	unsigned renderer_sel = 0;
+	GS_RENDERER renderer_setting = static_cast<GS_RENDERER>(theApp.GetConfig("Renderer", static_cast<int>(GS_DEFAULT_RENDERER)));
+	GS_RENDERER renderer_sel = GS_RENDERER::DX9_HW;
```

Comment:
Same as the other instance. Probably should be GS_RENDERER_DEFAULT. Again it probably doesn't make a difference using current implementations. It would just be cleaner to use GS_DEFAULT_RENDERER.